client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -153,33 +150,17 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 	}
 
 	renderActionTile() {
-		const { isConnected, siteUrl, translate } = this.props;
```

Comment:
For symmetry reasons, I'd kinda love if we could scrape the `renderActionTile` method altogether, and just inline `<ConnectIntro />` into the `{ hasBusinessAddress ? <ConnectSuccess /> : ... }` ternary.

Reply:
(Same for the contact form and stats steps, obviously.)

Reply:
Would be nice if all of those were consistent, but for the business address step this wouldn't work, because we have additional logic whether to show the intro screen or the form.

Reply:
In any case this looks like something for another PR, no?

Reply:
Yeah, okay.